### PR TITLE
equations for baselines

### PR DESCRIPTION
**Pull request by davharris**
If we like this, I can add a comment for the reviewers.  

Ideally, we should also discuss where the standard deviations of ε come from, but that differs for the observer-corrected and non-corrected "average" models so I can't figure out the best way to discuss it without breaking the flow.

### CODE REVIEW

**Comment by sdtaylor** [comment]
Observer models are already discussed prior to this, so I see no problem putting those details in this paragraph. Otherwise it's definitely an improvement. It might also help to include the ARIMA(0,1,0) notation for the naive model.

**Comment by davharris** [comment]
Okay, I've added more details and a line in the reviewer response. Might read awkwardly now, but it's probably good to have all this information available. Thanks @sdtaylor for your input above.

**Comment by sdtaylor** [comment]
LGTM

File: manuscript/draft.Rmd [hunk 1]
```diff
@@ -292,18 +292,35 @@ table_data %>%
...
+$\sigma^\mathrm{residual}$ from the observer model; see supplement).
+
+The second baseline, called the "naive" model [@hyndman2014], was a simple
+autoregressive process with a single year of history, i.e. an ARIMA(0,1,0)
+model: 
+
+$$y_t = y_{t-1} + \epsilon_t,$$
+
+where the standard deviation of $\epsilon$ is a free parameter for each site. In
+contrast to the "average" model, whose predictions were based on the average
```
Comment (ethanwhite) [inline manuscript/draft.Rmd]: since we say that the '"naive" model predicts', I think this line should be "whose predictions are based on"

**Comment by davharris** [comment]
Thanks both! Fixed the grammar, and merging now that it's been approved by both of you.

**Pull request merged**